### PR TITLE
Oj 1173 allure for common lambdas

### PR DESCRIPTION
**Pull request by gokhakan**
### What changed

We have added Allure reporting to this repo
### Why did it change

We did not have Allure previously

### Issue tracking

OJ-1173
https://govukverify.atlassian.net/jira/software/c/projects/OJ/boards/166?modal=detail&selectedIssue=OJ-1173&assignee=617fdbaaf485cd00680cb8be&assignee=60bdd0d58f2cc10069551099
### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ echo -e "\tCriIdentifier SSM key ${GREEN}$cri_identifier${NOCOLOR}"
 
 ./gradlew clean
 sam validate -t infrastructure/lambda/template.yaml
-sam build -t infrastructure/lambda/template.yaml
+sam build -t infrastructure/lambda/template.yaml --beta-features
```
Comment (techaman-m) [inline deploy.sh:36]: Do we need to commit this @jkunle  and @gokhakan

Comment (MattBidewell) [reply deploy.sh:36]: No we shouldnt need this at all

Comment (gokhakan) [reply deploy.sh:36]: I am removing this. It works without.

**Review by jkunle — APPROVED** [review]

File: accesstoken2/tsconfig.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "preserveConstEnums": true,
     // "noEmit": true, // emit for debugger testing purposes.
     "sourceMap": true,
-    "module": "es2015",
+    "module": "es2022",
```
Comment (MattBidewell) [inline accesstoken2/tsconfig.json:9]: Is this needed in this PR? is it related to the allure changes

**Review by MattBidewell — APPROVED** [review]

**Review by techaman-m — APPROVED** [review]
Looks very alluring to me 👍

**Review by merlinc — APPROVED** [review]

**Pull request merged**